merge-hf-and-lora-to-hf.py

```diff
@@ -0,0 +1,62 @@
+import torch
+import argparse
+from transformers import LlamaTokenizer, LlamaForCausalLM
+from peft import PeftModel
+
+# args with description.
+parser = argparse.ArgumentParser(
+    prog="Merge HF file with Lora\n",
+    description="Please locate HF format model path with pytorch_*.bin inside, lora path with adapter_config.json and adapter_model.bin.",
+)
+
+# The original base model checkpoint dir
+parser.add_argument(
+    "--model_path",
+    type=str,
+    default="decapoda-research/llama-7b-hf",
+    help="Directory contain original HF model",
+)
+# The finetuned lora model checkpoint dir
+parser.add_argument(
+    "--lora_path",
+    type=str,
+    default="decapoda-research/lora",
+    help="Directory contain Lora",
+)
+# The output dir
+parser.add_argument(
+    "--out_path",
+    type=str,
+    default="decapoda-research/lora-merged",
+    help="Directory store merged HF model",
+)
```

Comment:
Don't set a default, at least for the lora and out path. Otherwise running the script without parameters will try to do nonsense instead of showing help.  Explain that it is also possible to use a HF model name instead of a path.
```suggestion
# The original base model checkpoint dir
parser.add_argument(
    "--model",
    type=str,
    default="decapoda-research/llama-7b-hf",
    help="HF model name or path",
)
# The finetuned lora model checkpoint dir
parser.add_argument(
    "--lora",
    type=str,
    required=True,
    help="LoRA model name or path",
)
# The output dir
parser.add_argument(
    "--out",
    type=str,
    required=True,
    help="Directory store merged HF model",
)
```

Reply:
If you accept this suggestion as-is, remember to also update the names of the args when they are used later.

Reply:
Edited it in my phone😂